lib/faraday/http_cache.rb

```diff
@@ -322,8 +329,8 @@ def create_response(env)
       }
     end
 
-    def create_request(env)
-      Request.from_env(env)
+    def create_request(env, options = {})
+      Request.from_env(env, options)
```

Comment:
I believe we can revert these changes now that the serializer isn't being used in the Request class.